src/render/Cargo.toml

```diff
@@ -38,5 +38,5 @@ unstable = []
 cgmath = { version = "0.14", optional = true }
 derivative = "1.0"
 draw_state = "0.7"
-gfx_core = { path = "../core", version = "0.7.1" }
```

Comment:
I think it specifically requires `0.7.1`, so please don't remove this

Reply:
It doesn't have that effect though, only `=0.7.1` would have. `0.7.1` means error if not same or semver compatible newer. So it already locked at `0.7.2` and doesn't have much choice as that's what at that path.

I only changed it as specifying the min patch versions doesn't provide much and could be confusing.

Reply:
I don't think this is correct.
> only =0.7.1 would have

I don't need exactly 0.7.1, I need stuff introduced in 0.7.1.

> So it already locked at 0.7.2 and doesn't have much choice as that's what at that path.

It's only locked by some of the locks, including the one at the root of the repo. However, if there is a side crate that depends on gfx_render, and it has a lock file with "gfx_core-0.7.0", your new dependency will be satisfied, but it will fail to build.

For this reason, `cargo` may soon start picking the *smallest* compatible version number when testing a crate before publishing instead of the highest.

Reply:
Right, yeah I guess so! Thanks for explaining.